.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: CI
+
+on:
+  push:
+    
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  changed-files:
+    name: Check for changed files
+    runs-on: zeko-gha-runner
+    steps:
+      - name: 📥 Checkout
+        uses: actions/checkout@v5
+        with:
+          submodules: recursive
+      - name: ✨ Check for changed files
+        uses: tj-actions/changed-files@v46
+        id: changed
+        with:
+          files_yaml: |
+            zeko:
+              - 'src/app/zeko/**'
```

Comment:
not only if zeko files change, occasionally we are changing some of the core mina code